maintenance/action.yml

```diff
@@ -43,7 +43,7 @@ runs:
     - name: Enable maintenance mode
       if: inputs.mode == 'enable'
       shell: bash
-      run: make ${{ inputs.environment }} maintenance-fail-over
+      run: make ci ${{ inputs.environment }} maintenance-fail-over
```

Comment:
can we add this to line 53 as well please?

Reply:
done thanks